entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [[ ! -z ${1} ]] && [[ ${1} != "--config" ]]; then
+  exec "$@"
+elif [[ $(id -u) != 0 ]] && [[ $(id -g) != 0 ]]; then
```

Comment:
```suggestion
elif [[ $(id -u) != 0 ]] || [[ $(id -g) != 0 ]]; then
```

Reply:
Is this what would cause the tests to fail?

Reply:
hmmm, tests are passing now so I'm not sure if was anything wrong with the code. So I'll commit this